baybe/constraints/conditions.py

```diff
@@ -134,12 +174,14 @@ def evaluate(self, data: pd.Series) -> pd.Series:  # noqa: D102
                 "This operation is error-prone and not supported. Only use threshold "
                 "conditions with numerical parameters."
             )
-        func = rpartial(_threshold_operators[self.operator], self.threshold)
-        if self.operator in _valid_tolerance_operators:
-            func = rpartial(func, atol=self.tolerance)
-
+        func = self.generate_operator_function()
         return data.apply(func)
 
+    def to_polars(self, input: pl.Expr) -> pl.Expr:
```

Comment:
`input` is a python keyword. Probably better avoid

Reply:
changed to `expr`